input/pagecontent/index.md

```diff
@@ -56,6 +56,8 @@ The diagram below illustrates the interactions between the various actors:
 *Figure 1: FHIR RESTful interactions*
 <br/>
 
+*Note*: AU eRequesting Actor interactions will transition with the introduction of the Task resource, which will become central to fulfilment.
```

Comment:
I don't think this wording works anymore as the Task profile is included in the IG now.

Reply:
The wording has been confirmed with @michael-wilson-au, resolving this comment.

Reply:
Confirmed with @michael-wilson-au that wording remains appropriate as review of interactions has not yet taken place.